features/admin-history-pagination.feature

```diff
@@ -0,0 +1,30 @@
+Feature: Viewing a documents history
```

Comment:
I've tested the functionality via feature specs since it's a pretty e2e bit of functionality. Do people think it's enough coverage?

Reply:
Seems sensible to me